src/Symfony/Component/Notifier/Bridge/Slack/SlackTransport.php

```diff
@@ -69,22 +72,23 @@ protected function doSend(MessageInterface $message): void
         }
 
         $options = $opts ? $opts->toArray() : [];
-        $options['token'] = $this->accessToken;
-        if (!isset($options['channel'])) {
-            $options['channel'] = $message->getRecipientId() ?: $this->chatChannel;
-        }
+
         $options['text'] = $message->getSubject();
-        $response = $this->client->request('POST', 'https://'.$this->getEndpoint().'/api/chat.postMessage', [
-            'body' => array_filter($options),
-        ]);
+        $options['blocks'] = isset($options['blocks']) ? json_decode($options['blocks'], true) : null;
+
+        $response = $this->client->request(
+            'POST',
+            sprintf('https://%s/%s', $this->getEndpoint(), $this->path),
+            ['json' => array_filter($options)]
+        );
 
         if (200 !== $response->getStatusCode()) {
-            throw new TransportException(sprintf('Unable to post the Slack message: %s.', $response->getContent(false)), $response);
+            throw new TransportException(sprintf('Unable to post the Slack message: "%s".', $response->getContent(false)), $response);
```

Comment:
That's a false positive from fabbot (same for the other similar changes): we don't quote when the message comes from the outside and already formatter like this is the case here.